src/main/java/com/entropyteam/entropay/employees/services/ReportService.java

```diff
@@ -113,4 +117,21 @@ public Page<EmployeeReportDto> getEmployeesReport() {
         }
         return new PageImpl<>(employeesReportDtoList, Pageable.unpaged(), employeesReportDtoList.size());
     }
+
+    public List<Employee> applyFilters(ReactAdminParams params) {
+        if (params != null && !StringUtils.isEmpty(params.getFilter())) {
```

Comment:
We already have a mapper reading from `ReactAdminParams` and building a `Filter`: 
```
Filter filter = mapper.buildFilter(params, entityClass);
```
Then you can ask for the `active` field in `filter` object
You can see the implementation in `BaseService` class